realm/realm-jni/src/io_realm_internal_LinkView.cpp

```diff
@@ -176,3 +176,18 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_LinkView_nativeIsAttached
     } CATCH_STD()
     return 0;
 }
+
+JNIEXPORT jlong JNICALL Java_io_realm_internal_LinkView_nativeFind
+  (JNIEnv *env, jobject, jlong nativeLinkViewPtr, jlong rowIndex)
+{
+    TR_ENTER_PTR(nativeLinkViewPtr)
+    try {
+        LinkView *lv = LV(nativeLinkViewPtr);
+        if (!ROW_INDEX_VALID(env, lv, rowIndex)) {
+            return -1;
+        }
+        size_t ndx = lv->find(rowIndex);
+        return jlong(ndx);
```

Comment:
Thats what I had previously. @cmelchior @kneth what do you guys prefer me to do here?